### ISSUE TITLE
Database missing again

### ISSUE THREAD

**Issue by uwemdev** [uchimadara/PonziStory-2.0#1]
As usual 😴😴

**Comment by uchimadara** [comment]
This is a premium script.

I will set it up for you + database file for $50

**Comment by uchimadara** [comment]
message @ uchimadarasan@gmail.com to get it

**Comment by megdevs** [comment]
Nah, you mad a free project here and telling people to pay for it. Well, i fix the databased myself and it's working fine.

**Comment by uchimadara** [comment]
GoodLuck on that

**Comment by dharmajaya88** [comment]
Can i get database file ?

**Comment by uwemdev** [comment]
I got the db

**Comment by seriakioz** [comment]
database yok boşuna indirmeyin

**Comment by dongeneral02** [comment]
> message @ [uchimadarasan@gmail.com](mailto:uchimadarasan@gmail.com) to get it

I have messaged you sir.

**Comment by brashbeat** [comment]
Please share the database! We need it. Thank you!

**Comment by uwemdev** [comment]
Cost $50

**Comment by Talk2masterg** [comment]
Someone should help with the database 😕🙏🙏🙏
